docker/reference/reference_test.go

```diff
@@ -385,21 +385,21 @@ func TestSerialization(t *testing.T) {
 		}
 		b, err := json.Marshal(m)
 		if err != nil {
-			failf("error marshalling: %v", err)
+			failf("error marshaling: %v", err)
```

Comment:
I don't have a preferred spelling in this family, but it felt like this repo used this spelling more for this specific flavor (`-ing`).